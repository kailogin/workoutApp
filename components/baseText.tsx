import React from "react";
import { StyleSheet, Text, TextStyle } from "react-native";

import { Colors } from "../utils/colors";

interface BaseTextProps {
  style?: TextStyle;
}

export const BaseText: React.FC<BaseTextProps> = ({ children, style }) => {
  return <Text style={[styles.text, style]}>{children}</Text>;
};

const styles = StyleSheet.create({
  text: {
    color: Colors.WHITE,
    fontFamily: "Montserrat",
  },
});
