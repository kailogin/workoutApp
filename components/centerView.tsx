import { View } from "react-native";

interface CenterViewProps {}

export const CenterView: React.FC<CenterViewProps> = ({ children }) => {
  return <View>{children}</View>;
};
