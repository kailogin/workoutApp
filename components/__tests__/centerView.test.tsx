import "react-native";
import React from "react";
import renderer from "react-test-renderer";

import { CenterView } from "../centerView";
import { BaseText } from "../baseText";

const children = <BaseText>TEST ME </BaseText>;

test(`CenterView snapshot`, () => {
  const snap = renderer.create(<CenterView>{children}</CenterView>).toJSON();

  expect(snap).toMatchSnapshot();
});
