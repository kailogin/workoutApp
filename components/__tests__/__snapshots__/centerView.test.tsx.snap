// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CenterView snapshot 1`] = `
<View>
  <Text
    style={
      Array [
        Object {
          "color": "#fff",
          "fontFamily": "Montserrat",
        },
        undefined,
      ]
    }
  >
    TEST ME 
  </Text>
</View>
`;
